src/protocolgame.cpp

```diff
@@ -356,16 +372,31 @@ void ProtocolGame::onRecvFirstMessage(NetworkMessage& msg)
 		writeToOutputBuffer(opcodeMessage);
 	}
 
-	msg.skipBytes(1); // gamemaster flag
+	// Change packet verifying mode for QT clients
+	if (version >= 1111 && operatingSystem >= CLIENTOS_QT_LINUX && operatingSystem <= CLIENTOS_QT_MAC) {
```

Comment:
we aren't allowing <= 1111, so I guess this check is redundant and checksum mode should be sequence by default, which would also delete enum for CHECKSUM_SEQUENCE

Reply:
otclient compatible with this protocol still uses adler32

Reply:
well then I would say otc should be fixed to act like qt client.

Reply:
> Well then I would say otc should be fixed to act like qt client.

Is there development on their behalf? OTC shouldn't block TFS but it would be nice to have them move together.